### DISCUSSION TITLE
BLE Sensor Data to serial log

### DISCUSSION THREAD

**Discussion by iamhermes** [arendst/Tasmota#17466] [Q&A]
i have troubles to get every BLE sensor datagram to serial log.
is there some option i miss? 

Only periodic messages are published to serial with SENSOR prefix, but the sensor data is updated more often in poor reachable WiFi
i will not use wifi to collect the data (i think serial is better option for this)

need the BLE Messages direct to Serial

Program Version | 12.3.1.1(bluetooth)

wifi off
setoption115 1
MI32Option0 0
MI32Option1 0
MI32Option2 1

**Comment by sfromis** [comment]
I consider Wifi+MQTT **much** better for collecting data, this is what Tasmota is designed for.

You tell very little about what you see and what you lack, and not much about details of your setup, but in general the default serial logging in Tasmota is turned off 10 minutes after boot, unless you do something extra to keep it alive.
